lib/tlscontext.c

```diff
@@ -60,6 +60,8 @@ struct _TLSContext
   gchar *ca_file;
   gchar *cipher_suite;
   gchar *tls13_cipher_suite;
+  gchar *sigalgs;
```

Comment:
These should be freed ( in `_tls_context_free()`).

Reply:
Fixed, thanks.